test/analysis_test.cc

```diff
@@ -422,12 +436,25 @@ static void test_intersection_of_timestamps() {
 }
 
 void run_analysis_test() {
```

Comment:
I appreciate the effort to put in the text. There is a more idiomatic way of doing it by declaring a test function through a macro and that macro will be automatically print the name of the function when it is executed. Let's hold off these manually inserted output strings and I can change the tests to use the more idiomatic way.

Reply:
:+1: